### PR TITLE
fixed rebase problem, finished issue #2

### PR DESCRIPTION
**Pull request by bartist**

### CODE REVIEW

**Comment by bartist** [comment]
@steffansluis @WouterKvG Please check, sorry for the f-up

File: existing-communities/README.md [hunk 1]
```diff
@@ -1 +1,47 @@
-# Existing Communities
+# Communities
+A community is, according to the Oxford Dictionary,  *a group of people living in the same place or having a particular characteristic in common*{{"ComDef" | cite}}. In traditional research, the study focuses on localy bound communities like neighbourhoods and villages, because they have clear boundaries of who is involved in the community{{"meijering2007constructing" | cite}}{{"meijering2007intentional" | cite}}.  
+But thanks to modern media, like games, social media and messenger-applications, communities are not bound anymore by location and the characteristic becomes the central point of the community. A concequence of this is that boundaries of the community are less clear. Therefor, one of the challenges of doing research on these virtual communities is determining who is and who is not part of a community{{"sundaram2012understanding" | cite}}.
```
Comment (steffansluis) [inline existing-communities/README.md]: `Therefor, one of the challenges`

I think you mean therefore? http://grammarist.com/usage/therefore-vs-therefor/

File: existing-communities/README.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+But thanks to modern media, like games, social media and messenger-applications, communities are not bound anymore by location and the characteristic becomes the central point of the community. A concequence of this is that boundaries of the community are less clear. Therefor, one of the challenges of doing research on these virtual communities is determining who is and who is not part of a community{{"sundaram2012understanding" | cite}}.  
+Virtual communities have four necessary characteristics: interactivity, communicators, a publicly shared mediated (virtual) communication place and a minimum level of sustained membership{{"jones1997virtual" | cite}}. This means that in a virtual community, there must be a longlasting, selfsustaining communication *(interactivity)* about subjects (related to the relating characteristics) by two or more persons *(communicators)* over one ore more platform like a website or a chat-group *(communication place)* where there is a certain amount of active members *(sustained membership)*.
```
Comment (steffansluis) [inline existing-communities/README.md]: `one ore more platform` ==> `one or more platforms`

File: existing-communities/README.md [hunk 3]
```diff
@@ -1 +1,47 @@
...
+## Existing Communities
+Before we can compare different communities and their structure, we first need to define a few (types of) communities. We will look at two types of real-life communities and two types of virtual communities, such that we can discus their similarities and differences.
```
Comment (steffansluis) [inline existing-communities/README.md]: such that we can discus**s** their

File: existing-communities/README.md [hunk 4]
```diff
@@ -1 +1,47 @@
...
+### Ecosystems
+Ecosystems can be seen as a natural form of a community. It is bound to a location where a diversity of animals and species interact with eachother{{"EcosysDef" | cite}}. Although there is no real communication, the combination of individual self-interest (the survival of their offspring or themself) creates a network of dependencies and interactions that one can interpret as a community.
```
Comment (steffansluis) [inline existing-communities/README.md]: `It is bound` ==> `they are bound`

Edit: Maybe don't do this but singularize `Ecosystems` <- This

@bartist @wouterkvg: Ik neem (@steffansluis) taart mee maandag! :)

File: existing-communities/README.md [hunk 5]
```diff
@@ -1 +1,47 @@
...
+## Existing Communities
+Before we can compare different communities and their structure, we first need to define a few (types of) communities. We will look at two types of real-life communities and two types of virtual communities, such that we can discus their similarities and differences.
+### Ecosystems
+Ecosystems can be seen as a natural form of a community. It is bound to a location where a diversity of animals and species interact with eachother{{"EcosysDef" | cite}}. Although there is no real communication, the combination of individual self-interest (the survival of their offspring or themself) creates a network of dependencies and interactions that one can interpret as a community.  
+A key factor of an ecosystem is stability. In a stable ecosystem{{"EcosysSDef" | cite}}, each species acts in such a way that they, sometimes inderectly, enable or do not interfere with the survival of another species. An ecosystem will remain stable as long as the system is able to deal with outside disturbances.
```
Comment (steffansluis) [inline existing-communities/README.md]: `sometimes inderectly` ==> `sometimes indirectly`

Comment (steffansluis) [inline existing-communities/README.md]: `they, sometimes inderectly, enable or do not interfere`

This is a weirdly constructed sentence in my opinion. They are indirectly enabling and/or indirectly not interfering? I would rephrase it to something like: `each species acts in such way that they don't interfere with the survival of another species, they sometimes even enable it unintentionally`

File: existing-communities/README.md [hunk 6]
```diff
@@ -1 +1,47 @@
...
+
+### Intentional Communities
+Intentional communities are man-made communities, where its members attempt to realise a comen alternative way of life outside mainstream society{{"meijering2007intentional" | cite}}. Although there are no one list of criteria to characterise them, the most cited criteria are:
```
Comment (steffansluis) [inline existing-communities/README.md]: `realise a comen alternative` ==> `realise a common alternative`

Comment (steffansluis) [inline existing-communities/README.md]: outside  **of** mainstream society

Comment (steffansluis) [inline existing-communities/README.md]: there **is** no one list

Comment (steffansluis) [inline existing-communities/README.md]: `characterise`

Not sure whether we are using British or American spelling in other places...

File: existing-communities/README.md (hunk 1 above)
Comment (WouterKvG) [inline existing-communities/README.md]: I would change `But thanks to` to something like `With the introduction of`

File: existing-communities/README.md [hunk 7]
```diff
@@ -1 +1,47 @@
...
+
+1. No bonds by familial relationships only.
+2. A minimum of three to five adult members.
+3. Members join volunatarily
+4. Geographical and psychological seperation from mainstream society.
```
Comment (steffansluis) [inline existing-communities/README.md]: psychological sep**a**ration

File: existing-communities/README.md [hunk 8]
```diff
@@ -1 +1,47 @@
...
+### Ecosystems
+Ecosystems can be seen as a natural form of a community. It is bound to a location where a diversity of animals and species interact with eachother{{"EcosysDef" | cite}}. Although there is no real communication, the combination of individual self-interest (the survival of their offspring or themself) creates a network of dependencies and interactions that one can interpret as a community.  
+A key factor of an ecosystem is stability. In a stable ecosystem{{"EcosysSDef" | cite}}, each species acts in such a way that they, sometimes inderectly, enable or do not interfere with the survival of another species. An ecosystem will remain stable as long as the system is able to deal with outside disturbances.
+
+### Intentional Communities
+Intentional communities are man-made communities, where its members attempt to realise a comen alternative way of life outside mainstream society{{"meijering2007intentional" | cite}}. Although there are no one list of criteria to characterise them, the most cited criteria are:
+
+1. No bonds by familial relationships only.
+2. A minimum of three to five adult members.
+3. Members join volunatarily
```
Comment (steffansluis) [inline existing-communities/README.md]: `volunatarily` ==> `voluntarily`

File: existing-communities/README.md (hunk 2 above)
Comment (WouterKvG) [inline existing-communities/README.md]: `This means .. sustained membership` is a very long sentence. Consider splitting it.

File: existing-communities/README.md [hunk 9]
```diff
@@ -1 +1,47 @@
...
+4. Geographical and psychological seperation from mainstream society.
+5. A common ideology that is adhered to by all members.
+6. Sharing of (a part of) one's property.
+7. The interest of the group prevails over interests.
```
Comment (steffansluis) [inline existing-communities/README.md]: `prevails over interests`

I would say who's interests it prevails over, so `individual interests` or something like that

File: existing-communities/README.md [hunk 10]
```diff
@@ -1 +1,47 @@
...
+
+Not all intentional communities fulfill all of these criteria, as we will see later on, but they give a good idea of what is and what is not a intentional community. Intentional communities can be quiet diverse. To get a better understanding of the different ways an intentional community can work, they can be split into four groups: ecological, communal, religious and practical communities{{"meijering2007intentional" | cite}}.
```
Comment (steffansluis) [inline existing-communities/README.md]: `quiet diverse` ==> `quite diverse`

File: existing-communities/README.md [hunk 11]
```diff
@@ -1 +1,47 @@
...
+
+**Ecological Communities**  
+Ecological communities are communities that withdraw to remote locations in an effort to live according to their own ideals. They activly seperate themself from society and aim to be selfsufficient. The seperation is what identifies this type of intentional communities{{"meijering2007constructing" | cite}}{{"meijering2007intentional" | cite}}.
```
Comment (steffansluis) [inline existing-communities/README.md]: activ**e**ly seperate

File: existing-communities/README.md (hunk 3 above)
Comment (WouterKvG) [inline existing-communities/README.md]: `we first need to define the types of communities` is better I think (omit using `a few`).

File: existing-communities/README.md (hunk 11 above)
Comment (steffansluis) [inline existing-communities/README.md]: self**-**sufficient

Comment (steffansluis) [inline existing-communities/README.md]: sep**a**ration

File: existing-communities/README.md [hunk 12]
```diff
@@ -1 +1,47 @@
...
+
+**Communal Communities**  
+Communal communities center their ideals around the contact between communities members{{"meijering2007intentional" | cite}}. This means that the platform (chat-groups, website, etc.) or place (offices, pub, etc.) plays a central key in this type of community. The place or subject of the platform focuses on the particular characteristic that the members have in common, like a cooking-forum for people who like cooking or a chat-group for parents whos kids are in the same class.
+
+**Religious Communities**  
+Religious communities have their ideals cntered around a belief or religion. If you look at these communities based on the religions, then there is no boundry to the location of the community. However, there are religious communities that also behave like a communal community, like local church-communities. The ideals of the religion are the ideals of the community.
```
Comment (steffansluis) [inline existing-communities/README.md]: ideals c**e**ntered

File: existing-communities/README.md (hunk 10 above)
Comment (WouterKvG) [inline existing-communities/README.md]: `fulfill all of these criteria, as we will see later on, but they give a good idea` ==> `fulfill all of these criteria, but they give a good idea`

> Intentional communities can be **quite** diverse.

I would change the last sentence to: `Intentional communities can be split into four groups to get a better understanding of how they work: ecological etc..`

File: existing-communities/README.md (hunk 12 above)
Comment (steffansluis) [inline existing-communities/README.md]: `communities based on the religions`

Drop the `the` - Some actor in The Social Network

Comment (steffansluis) [inline existing-communities/README.md]: bound**a**ry

File: existing-communities/README.md [hunk 13]
```diff
@@ -1 +1,47 @@
...
+
+**Practical Communities**  
+Practical communities are special because they don't center around ideals at all. Practical communities arise out of pure convenience, when people are better of sharing information or resources (on a regulare basis). This can be a community of farmers sharing farming equipment, or students sharing a house.
```
Comment (steffansluis) [inline existing-communities/README.md]: `on a regulare basis` ==> `on a regular basis`

File: existing-communities/README.md [hunk 14]
```diff
@@ -1 +1,47 @@
...
+
+### MMO-RPG Communities
+Massive Multiplayer Online Role-Playing Games are, like the name describes, online games where the player explores a virtual world as one of many type of characters and can interact with an enormous amount of other players. Popular MMO-RPG's often have two types of communites: communities outside of the game (fans and forums) and communities inside of the game (parties and guilds). For this research, we will be looking at the communities inside the game, because the dynamics of these communities often require active participation.
```
Comment (steffansluis) [inline existing-communities/README.md]: one of many type**s**

File: existing-communities/README.md (hunk 11 above)
Comment (WouterKvG) [inline existing-communities/README.md]: > They actively **seperate themselves**

File: existing-communities/README.md (hunk 14 above)
Comment (steffansluis) [inline existing-communities/README.md]: inside **of** the game

File: existing-communities/README.md [hunk 15]
```diff
@@ -1 +1,47 @@
...
+
+### MMO-RPG Communities
+Massive Multiplayer Online Role-Playing Games are, like the name describes, online games where the player explores a virtual world as one of many type of characters and can interact with an enormous amount of other players. Popular MMO-RPG's often have two types of communites: communities outside of the game (fans and forums) and communities inside of the game (parties and guilds). For this research, we will be looking at the communities inside the game, because the dynamics of these communities often require active participation.  
+Most MMO-RPG's have two in-game systems that can be seen as communities: parties and guilds. A party is a small group of players that train or do quests (in-game missions) together for a short period of time (possibly one game session). The benefit of forming a party is experience-sharing (to some degree) and players can kill high-level in-game creatures they could kill on their own. If a party is formed for the purpose of doing a quest, the composition of the party (the distribution of roles or functions) may be addapted towards the goal of the quest to ensure optimal performance{{"chen2009communication" | cite}}.
```
Comment (steffansluis) [inline existing-communities/README.md]: `and players can kill high-level in-game creatures`

This doesn't really follow well after the previous part of the sentence. I would rephrase it to something like `and strength in number`

Edit: Also, if there are multiple benefits, it should be `benefits` instead of `benefit`

Comment (steffansluis) [inline existing-communities/README.md]: `addapted towards` ==> `adapted towards`

File: existing-communities/README.md [hunk 16]
```diff
@@ -1 +1,47 @@
...
+3. Members join voluntarily
+4. Geographical and psychological separation from mainstream society.
+5. A common ideology that is adhered to by all members.
+6. Sharing of (a part of) one's property.
+7. The interest of the group prevails over individual interests.
+
+Not all intentional communities fulfill all of these criteria but they give a good idea of what is and what is not a intentional community. Intentional communities can be quite diverse. To get a better understanding of the different ways an intentional community can work, they can be split into four groups: ecological, communal, religious and practical communities{{"meijering2007intentional" | cite}}.
+
+**Ecological Communities**  
+Ecological communities are communities that withdraw to remote locations in an effort to live according to their own ideals. They actively separate themself from society and aim to be self-sufficient. The separation is what identifies this type of intentional communities{{"meijering2007constructing" | cite}}{{"meijering2007intentional" | cite}}.
```
Comment (WouterKvG) [inline existing-communities/README.md]: > **The members** actively **seperate themselves**

Or, like before, make it singular again: `An ecological community is a community that withdraws to a remote location in an effort to live according to their own ideals. The members actively **seperate themselves**

File: existing-communities/README.md [hunk 17]
```diff
@@ -1 +1,47 @@
...
+Most MMO-RPG's have two in-game systems that can be seen as communities: parties and guilds. A party is a small group of players that train or do quests (in-game missions) together for a short period of time (possibly one game session). The benefit of forming a party is experience-sharing (to some degree) and players can kill high-level in-game creatures they could kill on their own. If a party is formed for the purpose of doing a quest, the composition of the party (the distribution of roles or functions) may be addapted towards the goal of the quest to ensure optimal performance{{"chen2009communication" | cite}}.  
+A guild is a group of players forming a community, for creating a social connection or to make it easier to form a trust-worthy or well-composed party over a longer period of time (longer than one game session){{"rossi2008mmorpg" | cite}}{{"kelly2004play" | cite}}{{"verhagen2009demystifying" | cite}}. Within a guild, members can be classified based on connectivity, type of player, role within the guild and involvement{{"ang2010social" | cite}}.
```
Comment (steffansluis) [inline existing-communities/README.md]: `community, for creating`

This isn't grammatically correct I think. It should be something like `community, with the purpose of` or the comma should be removed. Also, this sentence has too many `or`s in it in my opinion. I would split it into multiple sentences

File: existing-communities/README.md [hunk 18]
```diff
@@ -1 +1,47 @@
...
+7. The interest of the group prevails over individual interests.
+
+Not all intentional communities fulfill all of these criteria but they give a good idea of what is and what is not a intentional community. Intentional communities can be quite diverse. To get a better understanding of the different ways an intentional community can work, they can be split into four groups: ecological, communal, religious and practical communities{{"meijering2007intentional" | cite}}.
+
+**Ecological Communities**  
+Ecological communities are communities that withdraw to remote locations in an effort to live according to their own ideals. They actively separate themself from society and aim to be self-sufficient. The separation is what identifies this type of intentional communities{{"meijering2007constructing" | cite}}{{"meijering2007intentional" | cite}}.  
+Ecological communities can manifest themself on different scales. It can vary from a group living together in a house (cohousing) to a whole village (ecovillages){{"sanguinetti2012design" | cite}}, but all have a support-structure and cooperate to achieve the ideals of the community{"svensson2002ecovillage" | cite}.
```
Comment (WouterKvG) [inline existing-communities/README.md]: `Ecological communities can manifest themself` => `An ecological community can manifest itself`

File: existing-communities/README.md [hunk 19]
```diff
@@ -1 +1,47 @@
...
+A guild is a group of players forming a community, for creating a social connection or to make it easier to form a trust-worthy or well-composed party over a longer period of time (longer than one game session){{"rossi2008mmorpg" | cite}}{{"kelly2004play" | cite}}{{"verhagen2009demystifying" | cite}}. Within a guild, members can be classified based on connectivity, type of player, role within the guild and involvement{{"ang2010social" | cite}}.
+
+### Software Communities
+Software communities are groups of people involved in the development or improvement of a piece of software, but can also be focussed on a programing language (model). This makes software communities a broad term. For this research, we will be looking at Software Development Communities,  communities centered around software development/improvement, and we won't be looking at Software Engineering Communities,  communities centered around programing language (models), because the first community is our target group.
```
Comment (steffansluis) [inline existing-communities/README.md]: `For this research, we will be looking at Software Development Communities,  communities centered around software development/improvement, and we won't be looking at Software Engineering Communities,  communities centered around programing language (models), because the first community is our target group.`

This sentence should also be split up in my opinion. Also, the word `we` is used, which should be avoided.

File: existing-communities/README.md (hunk 18 above)
Comment (WouterKvG) [inline existing-communities/README.md]: > but all _ecological communites_ have a support-structure and co**-**operate

File: existing-communities/README.md [hunk 20]
```diff
@@ -1 +1,47 @@
...
+
+### Software Communities
+Software communities are groups of people involved in the development or improvement of a piece of software, but can also be focussed on a programing language (model). This makes software communities a broad term. For this research, we will be looking at Software Development Communities,  communities centered around software development/improvement, and we won't be looking at Software Engineering Communities,  communities centered around programing language (models), because the first community is our target group.   
+
+A software development communities can be devided into two groups: developers and users. Developers create software, wich is used by the users. The users are able to give feedback about the software, wich the developers can use in the next version of that piece of software.
```
Comment (steffansluis) [inline existing-communities/README.md]: `A software development communities can be devided into two groups` ==> `Software development communities can be divided into two groups`

File: existing-communities/README.md [hunk 21]
```diff
@@ -1 +1,47 @@
...
+Users sometimes discus the (lack of a certain) functionality in a software program on dedicated forums. This creates a feedback loop for the programmers. A smaller feedback loop is the bug-reporter. Organisations, like Apache and Mozilla, often have a part of their software dedicated to enabling the users to submit bugs{{"mockus2002two" | cite}}. This enables the developers to fix problems with the software in a much faster way than searching for feedback.
```
Comment (steffansluis) [inline existing-communities/README.md]: sometimes discus**s**

Comment (steffansluis) [inline existing-communities/README.md]: `Organisations`

British vs American spelling?

File: existing-communities/README.md [hunk 22]
```diff
@@ -1 +1,47 @@
...
+Users sometimes discus the (lack of a certain) functionality in a software program on dedicated forums. This creates a feedback loop for the programmers. A smaller feedback loop is the bug-reporter. Organisations, like Apache and Mozilla, often have a part of their software dedicated to enabling the users to submit bugs{{"mockus2002two" | cite}}. This enables the developers to fix problems with the software in a much faster way than searching for feedback.  
+One special community is the Opens Source Software community. Open Source means that the source-code of the software program is publicly available on the internet and that everyone can download and use the software for free{{"osterloh2007open" | cite}}. Because the code is publicly available, it enables users who understand the code to look in the code for the source of bugs and failures and give feedback related to a piece of the code. It also enables other programmers to expand the program based on it's source code.
```
Comment (steffansluis) [inline existing-communities/README.md]: `Opens Source Software` ==> `Open Source Software`

File: existing-communities/README.md [hunk 23]
```diff
@@ -1 +1,47 @@
...
+Ecological communities are communities that withdraw to remote locations in an effort to live according to their own ideals. They actively separate themself from society and aim to be self-sufficient. The separation is what identifies this type of intentional communities{{"meijering2007constructing" | cite}}{{"meijering2007intentional" | cite}}.  
+Ecological communities can manifest themself on different scales. It can vary from a group living together in a house (cohousing) to a whole village (ecovillages){{"sanguinetti2012design" | cite}}, but all have a support-structure and cooperate to achieve the ideals of the community{"svensson2002ecovillage" | cite}.
+
+**Communal Communities**  
+Communal communities center their ideals around the contact between communities members{{"meijering2007intentional" | cite}}. This means that the platform (chat-groups, website, etc.) or place (offices, pub, etc.) plays a central key in this type of community. The place or subject of the platform focuses on the particular characteristic that the members have in common, like a cooking-forum for people who like cooking or a chat-group for parents whos kids are in the same class.
```
Comment (WouterKvG) [inline existing-communities/README.md]: > The place or subject of the platform focuses on the particular characteristic that the members have in common, like a cooking-forum for people who like cooking or a chat-group for parents whos kids are in the same class.

==> 

> The place or subject of the platform focuses on the particular characteristic that the members have in common**. Examples of such communities are** a cooking-forum for people who like cooking **and** a chat-group for parents whos**e** kids are in the same class.

File: existing-communities/README.md [hunk 24]
```diff
@@ -1 +1,47 @@
...
+
+**Religious Communities**  
+Religious communities have their ideals centered around a belief or religion. If you look at these communities based on the religions, then there is no boundry to the location of the community. However, there are religious communities that also behave like a communal community, like local church-communities. The ideals of the religion are the ideals of the community.
```
Comment (WouterKvG) [inline existing-communities/README.md]: `If you look at these communities based on the religions, then there is no boundry to the location of the community` Watch out for mixing plural and singular forms in one sentence.
==>

> If you look at communities centered around a religion, then there are no boundaries to their location.

File: existing-communities/README.md [hunk 25]
```diff
@@ -1 +1,47 @@
...
+
+### Software Communities
+Software communities are groups of people involved in the development or improvement of a piece of software, but can also be focussed on a programing language (model). This makes software communities a broad term. For this research, we will be looking at Software Development Communities,  communities centered around software development/improvement, and we won't be looking at Software Engineering Communities,  communities centered around programing language (models), because the first community is our target group.   
+
+A software development communities can be devided into two groups: developers and users. Developers create software, wich is used by the users. The users are able to give feedback about the software, wich the developers can use in the next version of that piece of software.
```
Comment (WouterKvG) [inline existing-communities/README.md]: > A software development communit**y** can be d**i**vided
